assets/javascript/search.js

```diff
@@ -181,3 +187,20 @@ function showModalItem (itemId, itemStatus){
     });
 
 }
+
+
+function emailer(to_name, message_html) {
+
+    emailjs.init("user_ITZhTZz79nrtiOTftjpPw");
+
+    var template_params = {
+        "from_name": "tahreemsohailbutt@gmail.com",
+        "to_name": to_name,
+        "message_html": message_html
+    };
+
+    var service_id = "tahreemsohailbutt@gmail.com";
+    var template_id = "template_HXSdd43S";
+    //console.log("MEOWWW");
```

Comment:
can you remove this console.log statement from the code?

Reply:
removed now